shell/platform/darwin/ios/framework/Headers/FlutterCodecs.h

```diff
@@ -37,6 +37,7 @@ FLUTTER_EXPORT
                                arguments:(id)arguments;
 @property(readonly, nonatomic) NSString* method;
 @property(readonly, nonatomic) id arguments;
+- (BOOL)isEqualToMethodCall:(FlutterMethodCall*)other;
```

Comment:
That's what I've done, if you look in the implementation files. I was under the impression that adding isEqualToXxx to type Xxx was an ObjC standard and used across NSXxx classes. Guess I need to check my sources. I don't need to declare isEqual overrides in the header, do I?